oauth2_http/javatests/com/google/auth/oauth2/ServiceAccountCredentialsTest.java

```diff
@@ -1032,6 +1054,9 @@ static GenericJson writeServiceAccountJson(
     if (projectId != null) {
       json.put("project_id", projectId);
     }
+    if (quotaProjectId != null) {
```

Comment:
is it possible for the service account json file to have a null entry and is it worth writing a test case for it?

Reply:
It is, until this feature is live, this value will always be null. This is being implicitly tested via other tests that don't set this field. I am not sure I could add much more value with another test. Unless I am missing something?